VKSample/VKSample/Modules/FeedModule/FeedViewController.swift

```diff
@@ -115,14 +122,48 @@ final class FeedViewController: UIViewController {
         )
     }
 
-    private func setContentCellID(newsType: NewsType) -> String {
-        switch newsType {
-        case .post:
-            return Constants.postTextNibName
-        case .wallPhoto:
-            return Constants.postImageNibName
+    private func createPullRefresh() {
+        feedTableView.refreshControl = UIRefreshControl()
+        feedTableView.refreshControl?.tintColor = .gray
+        feedTableView.refreshControl?.addTarget(self, action: #selector(refreshAction), for: .valueChanged)
+    }
+
+    private func fetchLastNews() {
+        let firstNewsDate = news.first?.date ?? 0
+        vkNetworkService.fetchNewsfeed(startTime: firstNewsDate + 10) { [weak self] result in
```

Comment:
```suggestion
        vkNetworkService.fetchNewsfeed(startTime: firstNewsDate + ) { [weak self] result in
```
убираем магические числа